src/Faker/Provider/Image.php

```diff
@@ -7,17 +7,15 @@
  */
 class Image extends Base
 {
-    protected static $categories = array(
```

Comment:
Im afraid this is a BC break. Let's keep it and add a `@deprecation` notice.

Reply:
Added the notice